numpy/core/tests/test_umath.py

```diff
@@ -3439,13 +3433,14 @@ def check(x, rtol):
         x_series = np.logspace(-20, -3.001, 200)
         x_basic = np.logspace(-2.999, 0, 10, endpoint=False)
 
-        if dtype is np.longcomplex:
+        if glibc_older_than("2.19") and dtype is np.longcomplex:
```

Comment:
"2.19" or "2.18"?

Reply:
Not sure what that would do? If it's older than 2.18, ít's definitely older than 2.19?

Reply:
I was looking at the older comment that it was OK for glibc > 1.17.

Reply:
```
    skip_longcomplex_msg = ('Trig functions of np.longcomplex values known to be '
                            'inaccurate on aarch64 for some compilation '
                            'configurations, should be fixed by building on a '
                            'platform using glibc>2.17')
```

Reply:
For the if-branches here, the pertinent part is:
```
            # Can use 2.1 for > Ubuntu LTS Trusty (2014), glibc = 2.19.
            if skip_longcomplex_msg:
                pytest.skip(skip_longcomplex_msg)
```
I take from this comment that anything from glibc 2.19 onwards should be fine to take the `else`-clause (with the `2.1*eps`). I cannot very (easily or quickly) how far back in terms of glibc-versions things would work.

Reply:
> `[...] should be fixed by building on a platform using glibc>2.17'`

Right I couldn't evaluate that corner case (2.18), so I went with the conservative choice, of still running the check (resp. the looser tolerance) with 2.18.

Before the last commit, I actually had this as `glibc_older_than_2_17`, but then realized that this would do the wrong thing (due to `glibc > 2.17`) for manylinux 2014.

Reply:
OK, probably not worth worrying about. Manylinux2014 itself is getting toward end of life.